sa/storage-authority.go

```diff
@@ -296,6 +293,21 @@ func (ssa *SQLStorageAuthority) countCertificatesByName(domain string, earliest,
 	return len(serialMap), nil
 }
 
+// GetCertificateRequest looks up a certificate request by ID.
+func (ssa *SQLStorageAuthority) GetCertificateRequest(id string) (req core.CertificateRequest, err error) {
+	err = ssa.dbMap.SelectOne(&req, "SELECT * FROM certificateRequests WHERE id = :id",
+		map[string]interface{}{"id": id})
+	return
+}
+
+// GetLatestCertificateForRequest finds the certificate created under
+// the specificed certificate request that has the latest expiration.
+func (ssa *SQLStorageAuthority) GetLatestCertificateForRequest(requestID string) (cert core.Certificate, err error) {
+	err = ssa.dbMap.SelectOne(&cert, "SELECT * FROM certificates WHERE requestID = :requestID ORDER BY expires DESC LIMIT 0,1",
```

Comment:
I think it's confusing to give the offset (0) when nothing else in the SA does; please make this just `LIMIT 1`.